PINRemoteImage.podspec

```diff
@@ -19,12 +19,22 @@ Pod::Spec.new do |s|
 
   s.platform     = :ios, '6.0'
   s.requires_arc = true
+  
+  # Exclude optional FLAnimatedImage module
+  s.default_subspec = 'Core'
```

Comment:
@garrettmoon Yeah, I read up on it for hours lol so I'm quite familiar with the subspecs. I can make the default to include it. That makes a lot of sense.